man/standard-conf.xml

```diff
@@ -42,11 +42,12 @@
     <title>Configuration Directories and Precedence</title>
 
     <para>The default configuration is set during compilation, so configuration is only needed when it is
-    necessary to deviate from those defaults. Initially, the main configuration file in
-    <filename>/etc/systemd/</filename> contains commented out entries showing the defaults as a guide to the
-    administrator. Local overrides can be created by editing this file or by creating drop-ins, as described
-    below. Using drop-ins for local configuration is recommended over modifications to the main configuration
-    file.</para>
+    necessary to deviate from those defaults. The main configuration file is either in
+    <filename>/usr/lib/systemd</filename> or <filename>/etc/systemd/</filename> and contains commented out
+    entries showing the defaults as a guide to the administrator. Local overrides can be created by creating
+    drop-ins, as described below. The main configuration file can also be edited for this purpose (or a copy
+    in <filename>/etc</filename> if it's shipped in <filename>/usr</filename>) however using drop-ins for
```

Comment:
here too, slash is missing.

also, we do not ship "main config files" in /usr/, so it doesn't make sense to copy them over.